cumulus_etl/formats/deltalake.py

```diff
@@ -77,6 +77,8 @@ def initialize_class(cls, root: store.Root) -> None:
             cls.spark = delta.configure_spark_with_delta_pip(
                 builder,
                 extra_packages=[
+                    # See https://docs.delta.io/latest/delta-storage.html for advice
+                    # on which version of hadoop-aws to use.
```

Comment:
Unrelated, but I realized this was kind of a magic number unless you knew where to look.